internal/flags_test.go

```diff
@@ -275,7 +275,7 @@ func TestInitFlagsWithConfigMixedContent(t *testing.T) {
 		FeatureID:       getDefaultFlagValue(t, flagFeatureID),
 		ArtifactType:    getDefaultFlagValue(t, flagArtifactType),
 		ModuleType:      getDefaultFlagValue(t, flagModuleType),
-		StorageLocation: "",
+		StorageLocation: "./",
```

Comment:
Some other fields are initialized with getDefaultFlagValue. Why it's not used here?

Reply:
I'll use this function there too.